### DISCUSSION TITLE
Easystats Award for Students and Young Researchers (EASY-R) - 2025

### DISCUSSION THREAD

**Discussion by rempsyc** [easystats/easystats#435] [Award]
![image](https://github.com/user-attachments/assets/c3a6d83b-17a8-4bbc-b03f-da94ed3d08d3)

[_easystats_](https://easystats.github.io/easystats/) is an ecosystem of R packages designed to make statistical analyses easier, more accessible, and more transparent, empowering researchers to perform robust and interpretable analyses with minimal coding overhead. But it’s also more than just software—it’s a core team of seven dedicated maintainers deeply passionate about R and supported by a vibrant and growing community of contributors who share a passion for R and open-source development. We believe in the power of open science, the importance of transparency, and the potential of collaborative tools to democratize data analysis.

In that spirit, we’re excited to announce the _Easystats Award for Students and Young Researchers_ (EASY-R), which celebrates the next generation of researchers who harness the power of open-source tools to push the boundaries of statistical analysis and scientific knowledge. 

The award aims to encourage students and early-career researchers to embrace the values of open science, reproducibility, and methodological rigor. The award is 100% funded by [donations made to the _easystats_ project](https://opencollective.com/easystats), and we hope it will fuel and inspire those who show a commitment to transparency and accessibility in their work by recognizing exceptional papers that leverage _easystats_. It’s our way of fostering a community that values open-source tools, collaboration, and continuous learning in R and beyond—while celebrating the contributions of those who genuinely love _easystats_ and embody its principles.

## Instructions

- **Eligibility criteria:** Student or postdoc having first-authored a scientific paper using (and properly citing) one or more _easystats_ packages 
- **Selection criteria:** Strength of methodology and analyses, creative usage of _easystats_, dedication to open science (open methods, data, materials, access, preregistration, preprint), clarity and accessibility of writing
- **Deadline:** September 1st, 2025

The winner will be given the opportunity to **double the award** by making an _easystats_ contribution (e.g., contribute code, write a vignette, publish an online tutorial, hold a workshop, etc.)

## How to apply?

Submit a link (DOI) to your published paper in the thread below.

**Comment by rempsyc** [comment]
@easystats/maintainers forgot about this 👀 Should I close this conversation and make a new one for 2026?

**Reply by strengejacke** [reply]
Yes, and we need to promote it better
